hana/lib/HANAService.js

```diff
@@ -600,11 +600,11 @@ class HANAService extends SQLService {
 
         // Calculate final output columns once
         let outputColumns = ''
-        outputColumns = `_path_ as "_path_",${blobs} as "_blobs_",${expands} as "_expands_",${jsonColumn} as "_json_"`
+        outputColumns = `${this.quote('_path_')} as "_path_",${blobs} as "_blobs_",${expands} as "_expands_",${jsonColumn} as "_json_"`
```

Comment:
Why do we have to quote `_path_`? Isn't that a name chosen by us anyways?

Reply:
`_path_` is used as alias in CQN [here](https://github.com/cap-js/cds-dbs/blob/3e7dc2473e8298512f18dc07cb1d2402d7e8e3b8/hana/lib/HANAService.js#L419) hence also these snippets have to be quoted.